src/pages/Login/Login.scss

```
@@ -0,0 +1,148 @@
+@import '../../styles/mixin.scss';
+@import '../../styles/variables.scss';
+
+@mixin sectionHeaderSpan {
+  margin-bottom: 20px;
+  font-weight: bolder;
+  color: $var-black;
+}
+
+@mixin loginButton {
+  width: 100%;
+  padding: 10px 0;
+  border: 1px solid black;
+  background-color: white;
+  transition: all 0.2s ease-in-out;
+  cursor: pointer;
+}
+
+@mixin loginButtonHover {
+  color: white;
+  background-color: black;
+}
+
+.display {
+  opacity: 0;
+}
+
+.login {
+  @include flexContainer(center, center);
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.1);
+  font-size: 14px;
+  font-family: Arial, Helvetica, sans-serif;
+
+  .container {
+    @include flexContainer(flex-start, center);
+    width: 60%;
+    padding: 30px;
+    background-color: white;
+
+    .section-left {
+      width: 50%;
+      height: 100%;
+      margin-right: 30px;
+
+      .login-header {
+        span {
+          display: block;
+          margin-bottom: 10px;
+          color: $var-gray;
+        }
+        .left-section-header {
+          @include sectionHeaderSpan;
+        }
```

```suggestion
        span {
          display: block;
          margin-bottom: 10px;
          color: $var-gray;
        }
        
        .left-section-header {
          @include sectionHeaderSpan;
        }
```
가독성을 위해 선택자를 한 칸 띄워주세요~